template-skeleton/template/package.dev.json

```diff
@@ -28,6 +28,7 @@
     "@turf/boolean-intersects": "^6.3.0",
     "cypress": "^6.3.0",
     "deck.gl": "8.5.0-alpha.9",
+    "deepmerge": "^4.2.2",
```

Comment:
Why this?

Reply:
If a user (PS) wants to edit the theme, it is possible to do so without breaking it.
```js
cartoThemeOptions = {
  pallete: {
    main: '#fff
}

```